controllers/util/util.go

```diff
@@ -81,3 +86,13 @@ func AddLabel(obj metav1.Object, key string, value string) bool {
 	}
 	return false
 }
+
+func GetMD5Hash(text any) string {
```

Comment:
The argument name cannot be called text if you can accept `any`. Also, this function calculates the hash so a rename we'll needed
```suggestion
func calcMD5Hash(value any) string {
```

Reply:
Done